.github/workflows/maven-publish.yml

```diff
@@ -94,8 +94,8 @@ jobs:
             },
             {
               "id": "docker.io",
-              "username": "${{ secrets.DOCKER_HUB_USERNAME || ''}}",
-              "password": "${{ secrets.DOCKER_HUB_PASSWORD || ''}}"
+              "username": "${{ secrets.DOCKER_HUB_USERNAME || 'username'}}",
+              "password": "${{ secrets.DOCKER_HUB_PASSWORD || 'password'}}"
```

Comment:
I hadn't seen this but still fails.

Reply:
But we need to merge it no?

Reply:
This is why the HSC builds are failing

Reply:
Pushed this [commit](https://github.com/mekomsolutions/shared-github-workflow/commit/b8288fd3a8bb3c3742b5f4e8bb27fd899c9cf700), I hadn't seen this PR